Sources/App/Views/Error/ErrorPage.swift

```diff
@@ -0,0 +1,17 @@
+import Plot
+import Vapor
+
+
+final class ErrorPage: PublicPage {
+    let status: HTTPStatus
+
+    init(_ status: HTTPStatus) { self.status = status }
+
+    override func content() -> Node<HTML.BodyContext> {
+        .div(
+            // FIXME
+            .h1(.text("ERROR \(status)"))
+        )
+    }
```

Comment:
Here's where we can switch on error codes and return custom content.

Reply:
If you've got some copy/design I can add it to this PR. Or we merge and fill it in later? Let me know!